examples/step-62/step-62.cc

```diff
@@ -1317,11 +1317,10 @@ namespace step62
   template <int dim>
   void ElasticWave<dim>::run()
   {
-#ifdef DEBUG
-    pcout << "Debug mode" << std::endl;
-#else
-    pcout << "Release mode" << std::endl;
-#endif
+    if constexpr (library_build_mode == LibraryBuildMode::debug_build)
```

Comment:
I think this is a good idea - we need to move into the non-macro future. Can you pick a terser name? This would be easier to read if 'build' didn't show up three times.